setup.sh

```diff
@@ -76,7 +85,12 @@ cp templates/.project "$EXPORT_LOCATION/.project"
 cp templates/.gitattributes "$EXPORT_LOCATION/.gitattributes"
 print_success "  [OK]"
 
-print_info "  Copying Travis CI configuration..."
-sed "$REPLACE_TEMPLATE_VARS" templates/.travis.yml > "$EXPORT_LOCATION/.travis.yml"
+if [ "${BUILD_SERVICE}" == "github" ]; then
+  print_info "  Copying GitHub Actions configuration..."
+  sed "$REPLACE_TEMPLATE_VARS" templates/.travis.yml > "$EXPORT_LOCATION/.travis.yml"
```

Comment:
Am I crazy or the conditions are misplaced?

Reply:
Fixes. :)